en/websites_privacy_notice.md

```diff
@@ -31,45 +32,49 @@ We may use cookies, clear GIFs, third party web analytics, device information, a
 
 ---------------------------------------
 
-You can control individual cookie preferences, indicate your cookie preferences to others, and opt-out of web analytics and optimization tools. 
+You can control individual cookie preferences, indicate your cookie preferences to others, and opt out of web analytics and optimization tools. 
 {: #user-choices }
 
-* **Cookie History**: You can accept or decline individual cookies in your Firefox preferences in the Tools/Options/Privacy history section. Note that certain features of our products and services may not function properly without the aid of cookies. 
+* **Cookie History**: You can accept or decline individual cookies in your preferences in the Tools/Options/Privacy history section. Note that certain features of our products and services may not function properly without the aid of cookies. 
 {: #cookies }
 
-* **Do Not Track**: Mozilla does not track users across third party websites to provide targeted advertising. If you have configured your browser to send a “Do Not Track” signal when accessing our websites, Mozilla will not utilize any of the tools described in the [Metrics](https://www.mozilla.org/privacy/websites/#data-tools) section.
+* **Do Not Track**: Mozilla does not track users across third party websites to provide targeted advertising. In addition, if you have configured your browser to send a “Do Not Track” signal when accessing our websites, Mozilla will not utilize any of the tools described in the [Metrics](https://www.mozilla.org/privacy/websites/#data-tools) section.
+
+* **Email**: Our marketing communications are optional to receive and you can unsubscribe from the footer of the email or by updating your [Mozilla email preferences](https://www.mozilla.org/newsletter/recovery/), or for Thunderbird’s newsletter, on the [Thunderbird website](https://www.thunderbird.net/en-US/newsletter/).  
 
-* **Email**: Our marketing communications are optional to receive and you can unsubscribe from the footer of the email or by updating your [Mozilla email preferences](https://www.mozilla.org/newsletter/recovery/).
 
 * **Analytics & Optimization**: Follow the instructions below to prevent data collection about your visits to Mozilla websites:
-    * Mozilla first-party tools: [Opt-out on www.mozilla.org](https://www.mozilla.org/privacy/websites/data-preferences/).
+    * Mozilla first-party tools: [Opt out on www.mozilla.org](https://www.mozilla.org/privacy/websites/data-preferences/).
     * Google: Install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout), which prohibits data transmission to Google Analytics. Some Mozilla.org pages use clear GIFs which communicate with DoubleClick to understand the effectiveness of our advertising campaigns; you can control personalized ads from DoubleClick in Google's ad settings (you will be prompted to sign into your Google Account).
```

Comment:
@jejohnsonmoz 
- Smart quote in `Google's ad settings`
- Google account (lower case `a`)?

Reply:
Changed to lower case a. Couldn't find the right smart quote for an apostrophe.